src/samples/Authorization/AutoSignIn/appsettings.json

```diff
@@ -0,0 +1,54 @@
+{
+  "AgentApplication": {
+    "StartTypingTimer": true,
+    "RemoveRecipientMention": false,
+    "NormalizeMentions": false,
+
+    "UserAuthorization": {
+      "Default": "graph",
+      "AutoSignin": true,
+      "Handlers": {
+        "graph": {
+          "Settings": {
+            "AzureBotOAuthConnectionName": "graph"
+          }
+        }
+      }
+    }
+  },
+
+  "TokenValidation": {
```

Comment:
Can remove these TokenValidation settings?

Reply:
Not at this time yet.